frontend/src/app/core/services/user.service.ts

```diff
@@ -93,16 +102,12 @@ export class UserService {
     }
 
     public getZoomClientId(): Observable<string> {
-        return this.httpService
-            .getStringRequest(`${this.routePrefix}/zoom/client`)
-            .pipe(
-                tap({
-                    error: () =>
-                        this.notificationService.showErrorMessage(
-                            'Something went wrong. Failed to fetch zoom client id.',
-                        ),
-                }),
-            );
+        return this.httpService.getStringRequest(`${this.routePrefix}/zoom/client`).pipe(
+            tap({
+                error: () =>
+                    this.notificationService.showErrorMessage('Something went wrong. Failed to fetch zoom client id.'),
```

Comment:
we can use shared => constant => shared-messages

Reply:
fixed